web/settings.py

```diff
@@ -20,92 +24,81 @@
 # See https://docs.djangoproject.com/en/3.0/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = ')i@@^(m2b0jalyaa)r$2wg6o&mjb*rm_+cm9g03hyt=j61i2u('
+SECRET_KEY = ")i@@^(m2b0jalyaa)r$2wg6o&mjb*rm_+cm9g03hyt=j61i2u("
```

Comment:
This is a test key, right?

Reply:
It was auto-generated by Django when we created the project. I'm 100% planning on replacing it before we go-live, but it's okay to be there for now.